src/utils/displayTo.ts

```diff
@@ -1,33 +1,86 @@
+    (acc, next) => (next.breakpoints ? { ...acc, [next.screen]: formatQuery(next.breakpoints) } : acc),
+    {},
+  );
 }
 
-export function getMediaQueries(breakpoints: Breakpoints) {
-  if (!isBreakpoint(breakpoints, 'medium') || !isBreakpoint(breakpoints, 'large')) {
-    throw new Error('Invaild breakpoints');
+// create max-width media query including screen size
+export function displayUpTo(screen: ScreenSize) {
+  const screenBreakpoints = getScreenSizeBreakpoints(screen, breakpoints);
+  if (!screenBreakpoints) {
+    throw new Error('Invalid screen size');
   }
+  const query = formatQuery({ next: screenBreakpoints.next });
+  return formatMedia(query);
+}
 
-  return {
-    small: `(max-width: ${breakpoints.medium - 1}px)`,
-    smallAndMedium: `(max-width: ${breakpoints.large - 1}px)`,
-    medium: `(min-width: ${breakpoints.medium}px) and (max-width: ${breakpoints.large - 1}px)`,
-    mediumAndLarge: `(min-width: ${breakpoints.medium}px)`,
-    large: `(min-width: ${breakpoints.large}px)`,
-  };
+// create min-width media query including screen size
+export function displayFrom(screen: ScreenSize) {
+  const screenBreakpoints = getScreenSizeBreakpoints(screen, breakpoints);
+  if (!screenBreakpoints) {
+    throw new Error('Invalid screen size');
+  }
+  const query = formatQuery({ prev: screenBreakpoints.prev });
+  return formatMedia(query);
 }
 
 export function displayTo(screen: ScreenSize | string): ThemedCssFunction<any> {
-  const query = getMediaQueries(breakpoints)[screen] || screen;
-  const result: ThemedCssFunction<any> = (strings: any, ...interpolations: Array<any>) =>
+  const screenBreakpoints = getScreenSizeBreakpoints(screen as ScreenSize, breakpoints);
+  const query = screenBreakpoints ? formatQuery(screenBreakpoints) : screen;
+  return formatMedia(query);
+}
+
+function formatMedia(query: string): ThemedCssFunction<any> {
+  return (strings: any, ...interpolations: Array<any>) =>
     css`
       @media ${query} {
         ${css(strings, ...interpolations)};
       }
     `;
-  return result;
+}
+
+function getScreenSizeBreakpoints(
+  screen: ScreenSize,
+  breakpoints: Breakpoints,
+): { next?: number; prev?: number } | undefined {
+  switch (screen) {
+    case 'small':
+      return { next: breakpoints.medium };
+    case 'smallAndMedium':
+      return { next: breakpoints.large };
+    case 'medium':
+      return { prev: breakpoints.medium, next: breakpoints.large };
+    case 'mediumAndLarge':
+      return { prev: breakpoints.medium };
+    case 'large':
+      return { prev: breakpoints.large, next: breakpoints.xLarge };
+    case 'xLarge':
+      return { prev: breakpoints.xLarge, next: breakpoints.max };
+    case 'max':
+      return { prev: breakpoints.max };
+    default:
+      return undefined;
+  }
+}
+
+// create media query based on previous and next breakpoints
+export function formatQuery(breakpoints: { prev?: number; next?: number }) {
```

Comment:
You can skip 73 line
```suggestion
export function formatQuery({ next, prev }: { prev?: number; next?: number }) {
```

Reply:
Let the name of the variable `breakpoint` be visible